tests/bson/bug0592.phpt

```diff
@@ -44,15 +45,22 @@ object(stdClass)#%d (%d) {
   int(2147483647)
 }
 
+Test { "x": { "$numberLong": "4294967294" }}
+[%s] PHONGO-BSON: WARNING > Integer overflow detected on your platform: 4294967294
+object(stdClass)#%d (%d) {
+  ["x"]=>
+  string(10) "4294967294"
+}
+
```

Comment:
This test had missing output sections, and the warning formatting was wrong too. This test was always broken.